tests/cli_integration_test.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
+#![allow(clippy::uninlined_format_args)]
```

Comment:
Was this allow supposed to be here? I think most of them have been fixed. If not, clippy can fix them automatically I believe